syslog-ng-ctl/syslog-ng-ctl.c

```diff
@@ -243,6 +243,31 @@ slng_config(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
   return res;
 }
 
+static gboolean config_options_start = FALSE;
+static gboolean config_options_stop = FALSE;
+
+static GOptionEntry internal_options[] =
+{
+  { "start", 's', 0, G_OPTION_ARG_NONE, &config_options_start, "start live collection of internal logs", NULL },
+  { "stop", 'x', 0, G_OPTION_ARG_NONE, &config_options_stop, "stop live collection of internal logs", NULL },
+  { NULL,           0,   0, G_OPTION_ARG_NONE, NULL,                         NULL,           NULL }
+};
+
+static gint
+slng_internal(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
+{
+  GString *cmd = g_string_new("INTERLOGS ");
```

Comment:
Why do we concatenate strings here? We could as well use constant strings, which would be much more efficient.

Reply:
@jszigetvari : Yes, I will use the constant strings and remove this part